components/brave_rewards/browser/rewards_protocol_navigation_throttle_unittest.cc

```diff
@@ -8,10 +8,11 @@
 #include <tuple>
 #include <vector>
 
-#include "brave/components/brave_rewards/browser/rewards_protocol_handler.h"
+#include "brave/components/brave_rewards/browser/rewards_protocol_navigation_throttle.h"
 #include "testing/gtest/include/gtest/gtest.h"
 
-// npm run test -- brave_unit_tests --filter="*RewardsProtocolHandlerTest*"
+// npm run test -- brave_unit_tests
```

Comment:
Can we keep this in one line?

Reply:
I thought so, but you can add a `NOLINT` comment.

Reply:
`NOLINT` doesn't work for commented code.

Reply:
It should (unless something changed recently) — we have a couple instances.

Reply:
Sorry, I meant:
```cpp
// clang-format off
// npm run test -- brave_unit_tests --filter="*RewardsProtocolNavigationThrottleTest*"
// clang-format on
```
not a `NOLINT` comment.